### DISCUSSION TITLE
Add style only a particular column

### DISCUSSION THREAD

**Discussion by klebercarvalho** [TanStack/table#2258] [General] [answered]
I have a react-table where I need to add style only a particular column.
Is there any example where I do it?

Thank you

**Comment by klebercarvalho** [comment]
I found the solution here:

https://stackoverflow.com/questions/53605342/how-to-make-some-columns-align-left-and-some-column-align-center-in-react-table`

**Comment by martinschaer** [comment, marked as answer]
My solution is this:

```jsx
const columns = [{
  Header: 'Column 1',
  accessor: 'name',
  Cell: x => (<div className='text-center'>{x.cell.value}</div>)
},
...
]
```

**Reply by klebercarvalho** [reply]
Thank you @martinschaer
